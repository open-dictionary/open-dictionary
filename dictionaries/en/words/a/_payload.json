[{"data":1,"prerenderedAt":32},["Reactive",2],{"Kfx9XwALJ9":3},[4,24],{"language":5,"content":6},"en",{"word":7,"definitions":8},"a",[9,12,14,16,18,20,22],{"definition":10,"pos":11},"a metric unit of length equal to one ten billionth of a meter (or 0.0001 micron)","noun",{"definition":13,"pos":11},"any of several fat-soluble vitamins essential for normal vision",{"definition":15,"pos":11},"one of the four nucleotides used in building DNA",{"definition":17,"pos":11},"(biochemistry) purine base found in DNA and RNA",{"definition":19,"pos":11},"the basic unit of electric current adopted under the Systeme International d'Unites",{"definition":21,"pos":11},"the 1st letter of the Roman alphabet",{"definition":23,"pos":11},"the blood group whose red cells carry the A antigen",{"language":25,"content":26},"fa",{"word":7,"definitions":27},[28,30],{"definition":29},"حرف اول الفباى انگليسى",{"definition":31},"حرف اضافه مثبت",1690956235679]