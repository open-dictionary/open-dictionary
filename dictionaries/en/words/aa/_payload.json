[{"data":1,"prerenderedAt":14},["Reactive",2],{"JhiMPQUNzs":3},[4],{"language":5,"items":6},"en",[7,10,12],{"definition":8,"pos":9},"a dry form of lava resembling clinkers","noun",{"definition":11,"pos":9},"an international organization that provides a support group for persons trying to overcome alcoholism",{"definition":13,"pos":9},"an associate degree in arts",1690666540799]