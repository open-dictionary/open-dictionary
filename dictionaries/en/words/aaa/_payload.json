[{"data":1,"prerenderedAt":12},["Reactive",2],{"gJlwwaHj8A":3},[4],{"language":5,"content":6},"en",{"word":7,"definitions":8},"aaa",[9],{"definition":10,"pos":11},"an aneurysm of the abdominal aorta associated with old age and hypertension","noun",1690956244826]