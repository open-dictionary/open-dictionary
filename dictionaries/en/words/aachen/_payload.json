[{"data":1,"prerenderedAt":10},["Reactive",2],{"I1rYnByWOL":3},[4],{"language":5,"items":6},"en",[7],{"definition":8,"pos":9},"a city in western Germany near the Dutch and Belgian borders","noun",1690619309412]