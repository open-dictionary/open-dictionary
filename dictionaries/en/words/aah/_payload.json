[{"data":1,"prerenderedAt":12},["Reactive",2],{"zUMGe5WgoG":3},[4],{"language":5,"content":6},"en",{"word":7,"definitions":8},"aah",[9],{"definition":10,"pos":11},"express admiration and pleasure by uttering `ooh' or `aah'","verb",1690956244829]