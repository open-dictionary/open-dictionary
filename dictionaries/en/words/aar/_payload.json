[{"data":1,"prerenderedAt":12},["Reactive",2],{"TSyXLvg71E":3},[4],{"language":5,"content":6},"en",{"word":7,"definitions":8},"aar",[9],{"definition":10,"pos":11},"a river in north central Switzerland that runs northeast into the Rhine","noun",1690969955128]