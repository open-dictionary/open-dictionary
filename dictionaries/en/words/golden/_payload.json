[{"data":1,"prerenderedAt":36},["Reactive",2],{"NmmYXQysTZ":3},[4,22],{"language":5,"content":6},"en",{"word":7,"definitions":8},"golden",[9,12,14,16,18,20],{"definition":10,"pos":11},"having the deep slightly brownish color of gold","adjective",{"definition":13,"pos":11},"marked by peace and prosperity",{"definition":15,"pos":11},"made from or covered with gold",{"definition":17,"pos":11},"supremely favored",{"definition":19,"pos":11},"suggestive of gold",{"definition":21,"pos":11},"presaging or likely to bring good luck or a good outcome",{"language":23,"content":24},"fa",{"word":7,"definitions":25},[26,28,30,32,34],{"definition":27},"طلائى",{"definition":29},"طلايى",{"definition":31},"زرين",{"definition":33},"اعلا",{"definition":35},"درخشنده",1690956356109]